src/Queue.php

```diff
@@ -9,15 +9,16 @@
 namespace Yiisoft\Yii\Queue;
 
 use Psr\EventDispatcher\EventDispatcherInterface;
-use Psr\Log\LoggerInterface;
-use Yiisoft\EventDispatcher\Dispatcher;
 use Yiisoft\EventDispatcher\Provider\Provider;
```

Comment:
`yiisoft/event-dispatcher` is not found in composer.json

Reply:
It ms added in #29